readalongs/cli.py

```diff
@@ -212,8 +212,8 @@ def cli():
     multiple=True,
     callback=joiner_callback(LANGS),
     help=(
-        "The language code(s) for text in TEXTFILE (use only with -i, i.e., with plain text input); "
-        "multiple codes can be joined by ':' or by repeating the option; "
+        "The language code(s) for text in TEXTFILE (use only with plain text input); "
+        "multiple codes can be joined by ':', or by repeating the option, to enable the g2p cascade; "
```

Comment:
Link to g2p cascade docs here

Reply:
Yeah, that would be important. I'm not quite sure how best to do it...

I could link to readthedocs, but I don't really like that to be the main source. I'm inclined to put a paragraph about the g2p cascade in the `def align` docstring, so it's in the body of `g2p align -h`, refer to that in the `-l` switch doc for both `align` and `prepare`.

Hmm, I see I documented the cascade in `readalongs g2p -h`. I'll need to think this through more carefully and do something coherent in `g2p`, `prepare` and `align`... Maybe all I need is something like `(run 'readalongs g2p -h' for details)`.